scoreboard/tests/unit.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
Can you rename this to scoreboard/tests/utils_test.py?  I want to keep tests for each python module separate.